### ISSUE TITLE
Ansible-galaxy. Can't download the role

### ISSUE THREAD

**Issue by danylevskyi** [geerlingguy/ansible-role-php#87]
Here is the output:
```
$ sudo ansible-galaxy install geerlingguy.php
- downloading role 'php', owned by geerlingguy
https://galaxy.ansible.com/roles/432/versions/?page=2&page_size=50
Traceback (most recent call last):
  File "/usr/local/bin/ansible-galaxy", line 959, in <module>
    main()
  File "/usr/local/bin/ansible-galaxy", line 953, in main
    fn(args, options, parser)
  File "/usr/local/bin/ansible-galaxy", line 819, in execute_install
    if len(role_versions) > 0:
TypeError: object of type 'NoneType' has no len()
```

At the same time other roles, like `geerlingguy.mysql` are being installed properly.

**Comment by Josh-Campbell** [comment]
I get the same error. As a work-around I'm installing the previous version:
  ansible-galaxy install geerlingguy.php,1.9.1

**Comment by geerlingguy** [comment]
See upstream issue: https://github.com/ansible/galaxy-issues/issues/134

Currently, the only fix is to run Ansible 2.0.1+; the fix is in the 1.9-stable branch, but won't be in a release until 1.9.5 or a 1.9.4.1 (something like that).
